modules/clock.js

```diff
@@ -0,0 +1,10 @@
+import { time } from './variable.js';
+// eslint-disable-next-line import/no-unresolved
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayClock() {
+  setInterval(() => {
+    const timeNow = DateTime.now().toFormat('yyyy-MM-dd HH:mm:ss');
+    time.innerHTML = timeNow;
+  }, 1);
+}
```

Comment:
- [ ] Ouch! this is close but the current date is not being displayed on the webpage and this could be because you have added the `package.json` files in the `.gitignore` which means we can't download the proper packages so please make sure to remove them from the `.gitignore`.
- [ ] Additionally as part of the project requirements please make sure to use ES6 arrow functions instead of function declarations so please update it at line `5`.
- [ ] Furthermore, it's important to avoid manually disabling linter checks using inline code. Doing so can lead to errors and make it harder to maintain code quality. So please remove the code at line `2`. ✔